src/FormattingToolbar/index.js

```diff
@@ -368,61 +368,89 @@ export default class FormatToolbar extends React.Component {
     const selectedInlineHref = document.getClosestInline(selection.anchor.path);
     const selectedText = this.props.editor.value.document
       .getFragmentAtRange(this.props.editor.value.selection).text;
-
+    const calculateTruncatedLinkText = (text) =>{
+      let truncatedText;
+      try{
+        truncatedText = text.slice(0,20)+(text.length>20?'...':'');
+      }catch(_){
+        truncatedText = 'link';
+      }
+      return truncatedText;
+    }
```

Comment:
I believe this can be done with [CSS alone](https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow)

Reply:
Sure! Thank you so much for sharing the resource.